polling_stations/settings/static_files.py

```diff
@@ -9,8 +8,7 @@
     ],
     extra_js=[],
 )
-PIPELINE['JS_COMPRESSOR'] = 'pipeline.compressors.uglifyjs.UglifyJSCompressor'
-PIPELINE['UGLIFYJS_BINARY'] = 'node_modules/uglify-js/bin/uglifyjs'
```

Comment:
`jsmin` seems to work fine here :+1: You can also remove the uglify dependency from `package.json` here https://github.com/DemocracyClub/UK-Polling-Stations/blob/master/package.json#L12 if we're going to use something else.